freqtrade/optimize/backtesting.py

```diff
@@ -476,6 +539,21 @@ def _enter_trade(self, pair: str, row: List) -> Optional[LocalTrade]:
                 buy_tag=row[BUY_TAG_IDX] if has_buy_tag else None,
                 exchange='backtesting',
             )
+            order = Order(
+                ft_is_open=False,
+                ft_pair=trade.pair,
+                symbol=trade.pair,
+                ft_order_side="buy",
+                side="buy",
+                order_type="market",
+                status="closed",
+                price=trade.open_rate,
+                average=trade.open_rate,
+                amount=trade.amount,
+                cost=trade.stake_amount + trade.fee_open
+            )
+            trade.orders = []
```

Comment:
I don't think this should be necessary - `orders` is initialized as a list in the Object already (unless you encountered some problem with that?)

Reply:
It was needed because for some reason trade objects were otherwise using the same list (backtesting added orders from multiple pairs into the same list). Setting it into a new one resolved it.

Reply:
i've kindof suspected this ... 
in that case i think this should be set as part of the constructor (`__init__` ) of the `LocalTrade` object.

Reply:
Yes, moved the orders=[] into construct of trade